tests/helpers/test_translation.py

```diff
@@ -64,10 +64,16 @@ def test_load_translations_files_by_language(
             "test": {
                 "entity": {
                     "switch": {
-                        "other1": {"name": "Other 1"},
+                        "other1": {
+                            "name": "Other 1",
+                            "native_unit_of_measurement": "Units",
```

Comment:
I think units should generally not be capitalised, so I suggest that you amend the examples accordingly:
```suggestion
                            "native_unit_of_measurement": "units",
```

(Please adjust all below also...)